GoogleForCreators/web-stories-wp
# Templates: Renames template directories

## Context

To accommodate the increasing number of templates, we may consider restructuring how the templates are loaded.

## Summary

- Renames vertical based directories for raw templates and stickers, to use title from the first page of the story template.
- Updates media asset paths to use new changes introduced in #7695 
- Updates templates package tests for new changes.

## Relevant Technical Choices

This changes also prompted to make changes to the way all templates were loaded, raw template json will now be placed under `packages/templates/src/raw/<template_name>` directory. `<template_name>` will be the title shown on the first page of the story template.

```
-- <template_name>
  | --- index.js
  | --- template.json
```

`index.js` is where the template object is exported from and has definition as shown below. 

```js

/**
 * Internal dependencies
 */
import { default as template } from './template';

export default {
  title: '',
  tags: [],
  colors: [],
  description:'',
  ...template,
  vertical: '',
};
```

## To-do

n/a

## User-facing changes

n/a

## Testing Instructions

- Navigate to 'Explore Templates' on admin side.
- Verify that all eight story templates are present and working as expected.

### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #
